### PR TITLE
ISS1 - grouped functions in separate modules

### LINKED ISSUE #1: ISS1 - Refactor file structure
Files in repo are unordered in proper folders. Reorder them so file structure represent project structure

### PR DESCRIPTION
**Pull request by Recokioshi**

### CODE REVIEW

File: router/router.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	"log"
+	"net/http"
+	"os"
+)
+
+// RootRouter is router that binds passed handler functions to the root path
+func RootRouter(handler http.HandlerFunc) {
+	http.HandleFunc("/", handler)
+
+	port := os.Getenv("PORT")
```
Comment (Siergus93) [inline router/router.go:13]: Tell me whyyyyyyyy!
once you assign value to variable 'port' outside if, and later, you do assigment to variable in if body, this 'err' variable

**Review by Siergus93 — CHANGES_REQUESTED** [review]

File: router/router.go (hunk 1 above)
Comment (Recokioshi) [reply router/router.go:13]: initial assignment ment in the if statement has scope limited to if statement body. I need the port variable after if body for ListenAndServe func so I can't assign port as an initial assignment in its if. I don't need err outside its if body tho, so I can assign it inline as initial variable

Comment (Siergus93) [reply router/router.go:13]: Fine... xD

**Review by Siergus93 — APPROVED** [review]

**Review by Siergus93 — APPROVED** [review]
Fine by me

**Review by Siergus93 — APPROVED** [review]

**Review by Siergus93 — APPROVED** [review]
blablabla

**Review by Siergus93 — APPROVED** [review]

**Pull request merged**